src/utils/entitySpec.js

```diff
@@ -32,6 +41,67 @@ export default class EntitySpec {
     return new EntitySpec(type, provider, namespace, name, revision, pr)
   }
 
+  static checkDroppedFiles(files) {
+    let acceptedFiles = []
+    let rejectedFiles = []
+
+    files.forEach(file => {
+      const acceptedType = acceptedFilesValues.find(el => el === file.type)
+
+      if (acceptedType) acceptedFiles.push(file)
+      else rejectedFiles.push(file)
+    })
+
+    return {
+      acceptedFiles,
+      rejectedFiles
+    }
+  }
+
+  static fromUrl(url) {
+    const urlObject = new URL(url)
+    const pathname = urlObject.pathname.startsWith('/') ? urlObject.pathname.slice(1) : urlObject.pathname
+    const [packageName, name, version, revision] = pathname.split('/')
```

Comment:
I'm not so sure about this implementation.
Different providers have different elements, so i think it's better to have separate split functions, or either a separate and reusable function.
Thoughts?